packages/devtools_test/pubspec.yaml

```diff
@@ -0,0 +1,53 @@
+name: devtools_test
+description: A package containing shared test helpers for Dart DevTools tests.
+
+# Note: this version should only be updated by running tools/update_version.dart
+# that updates all versions of packages from packages/devtools.
+# When publishing new versions of this package be sure to publish a new version
+# of package:devtools as well. package:devtools contains a compiled snapshot of
+# this package.
+version: 2.6.1-dev.2
+
+homepage: https://github.com/flutter/devtools
+
+environment:
+  sdk: '>=2.10.0 <3.0.0'
+  # The flutter desktop support interacts with build scripts on the Flutter
+  # side that are not yet stable, so it requires a very recent version of
+  # Flutter. This version will increase regularly as the build scripts change.
+  flutter: '>=1.10.0'
+
+dependencies:
+  async: ^2.0.0
+  collection: ^1.15.0-nnbd
+  dds: ^2.0.1
+  devtools_shared: 2.6.1-dev.2
+  devtools_app: 2.6.1-dev.2
+  flutter:
+    sdk: flutter
+  meta: ^1.3.0
+  path: ^1.8.0
+  pedantic: ^1.11.0
+  provider: ^5.0.0
+  sse: ^4.0.0
+  vm_service: ^7.1.1
+  vm_snapshot_analysis: ^0.6.0
+  web_socket_channel: ^2.1.0
+
+dev_dependencies:
+  build_runner: ^2.0.4
+  flutter_test:
+    sdk: flutter
+  mockito: ^5.0.9
+  webkit_inspection_protocol: '>=0.5.0 <2.0.0'
+
+dependency_overrides:
+  # The '#OVERRIDE_FOR_DEVELOPMENT' lines are stripped out when we publish.
+  # All overriden dependencies are published together so there is no harm
+  # in treating them like they are part of a mono-repo while developing.
+  devtools_app: #OVERRIDE_FOR_DEVELOPMENT
+    path: ../devtools_app #OVERRIDE_FOR_DEVELOPMENT
+  devtools_server: #OVERRIDE_FOR_DEVELOPMENT
```

Comment:
we can remove this, as we don't depend on devtools_server above

Reply:
Thanks! Removed this (as well as other unused deps in this file).